src/EnergyPlus/Pumps.hh

```diff
@@ -122,131 +113,93 @@ namespace Pumps {
     {
         // Members
         std::string Name;
-        std::string ManualRPMSchedName;
-        int ManualRPMSchedIndex;
-        std::string LowerPsetSchedName;
-        int LowerPsetSchedIndex;
-        std::string UpperPsetSchedName;
-        int UpperPsetSchedIndex;
-        std::string MinRPMSchedName;
-        int MinRPMSchedIndex;
-        std::string MaxRPMSchedName;
-        int MaxRPMSchedIndex;
-        ControlTypeVFD VFDControlType; // VFDControlType
-        Real64 MaxRPM;                 // Maximum RPM range value - schedule limit
-        Real64 MinRPM;                 // Minimum RPM range value - schedule limit
-        Real64 PumpActualRPM;          // RPM recalculated from final flow through the loop
-
-        // Default Constructor
-        PumpVFDControlData()
-            : ManualRPMSchedIndex(0), LowerPsetSchedIndex(0), UpperPsetSchedIndex(0), MinRPMSchedIndex(0), MaxRPMSchedIndex(0),
-              VFDControlType(ControlTypeVFD::Invalid), MaxRPM(0.0), MinRPM(0.0), PumpActualRPM(0.0)
-        {
-        }
+        int ManualRPMSchedIndex = 0;
+        int LowerPsetSchedIndex = 0;
+        int UpperPsetSchedIndex = 0;
+        int MinRPMSchedIndex = 0;
+        int MaxRPMSchedIndex = 0;
+        ControlTypeVFD VFDControlType = ControlTypeVFD::Invalid; // VFDControlType
+        Real64 MaxRPM = 0.0;                                     // Maximum RPM range value - schedule limit
+        Real64 MinRPM = 0.0;                                     // Minimum RPM range value - schedule limit
+        Real64 PumpActualRPM = 0.0;                              // RPM recalculated from final flow through the loop
```

Comment:
@amirroth requested this and I ran with it.  The only benefit I see with keeping the schedule name is for debugging, but that's of limited value I think.

Reply:
No, caching schedule names was not a good idea. It has not been discussed in my recollection. PS There are multiple other instances of this type of thing. Too much string copying in general in EnergyPlus.

Reply:
Thanks for the replies! (I view this as an opportunity to learn)

I agree there's too much caching generally.

For debugging you should be able to do state->Xxxx anyways so that's not a concern.